### DISCUSSION TITLE
Support for other video games

### DISCUSSION THREAD

**Discussion by kiweezi** [timvisee/lazymc#73] [General]
Hey! Big fan of the project, thanks for taking your time to create it!
You've basically made Minecraft servers compatible with serverless architectures.

Is it possible that the high-level methods you used here can be applied to other dedicated video game servers, outside of Minecraft?
I'm not referring to you actually doing that, but just wondered if it was possible to do this with other games too.
E.g. Rust, Valheim, Terraria etc.

I took a look through your code, but I don't have much knowledge in the area, so thought I would trust the genius themselves!

**Comment by timvisee** [comment]
I've no experience with the protocols used in the other games. But assuming they're similar: probably!
